client/packages/android/gradle.properties

```diff
@@ -20,5 +20,3 @@ org.gradle.jvmargs=-Xmx1536m
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
```

Comment:
nice ! (I think this was a property that was making it hard for me to use capacitor, i.e when running `cap sync`)